### PR TITLE
Feat: Previous and Next buttons work.

### PR DESCRIPTION
**Pull request by JaSNOHo**
The cheeps are updated for each page.
No warnings or errors.
Can change page from both the URL and buttons.
Line 44-46 in Public.cshtml are inspired by W3Schools

Co-authored-by: Jasmin <jsho@itu.dk>

### CODE REVIEW

File: src/ChirpInfrastructure/CheepRepository.cs [hunk 1]
```diff
@@ -24,18 +24,22 @@ public class CheepRepository(ChirpDBContext context) : ICheepRepository
...
+	int pageSize = 32;
 	//query for getting every cheep
 	var query = _context.Cheeps.Select(message => new CheepDTO() // message = domain cheep. result = cheepDTO
 	{
 	  Text = message.Text,
 	  AuthorID = message.AuthorId,
 	  AuthorName = message.Author.Name,
 	  TimeStamp = message.TimeStamp.ToString("MM/dd/yy H:mm:ss") 
-	}).OrderBy(message => message.AuthorID); // As long as TimeStamp.ToString("") is present then we can't sort cheeps by time.
+	})
```
Comment (Thyme-Cat) [inline src/ChirpInfrastructure/CheepRepository.cs:38]: Is there a particular reason for this line break?

File: src/ChirpInfrastructure/CheepRepository.cs [hunk 2]
```diff
@@ -24,18 +24,22 @@ public class CheepRepository(ChirpDBContext context) : ICheepRepository
 
   }
   Above will be relevant later*/
-  public List<CheepDTO> ReadCheeps()
+  public List<CheepDTO> ReadCheeps(int pageNumber)
   {
-	var cheepList = new List<CheepDTO>();
-
+	//var cheepList = new List<CheepDTO>();
```
Comment (tomat-suppe) [inline src/ChirpInfrastructure/CheepRepository.cs:29]: Do we need to remove this in the future?

File: src/ChirpInfrastructure/CheepRepository.cs [hunk 3]
```diff
@@ -24,18 +24,22 @@ public class CheepRepository(ChirpDBContext context) : ICheepRepository
...
+	int pageSize = 32;
 	//query for getting every cheep
 	var query = _context.Cheeps.Select(message => new CheepDTO() // message = domain cheep. result = cheepDTO
 	{
 	  Text = message.Text,
 	  AuthorID = message.AuthorId,
 	  AuthorName = message.Author.Name,
 	  TimeStamp = message.TimeStamp.ToString("MM/dd/yy H:mm:ss")
```
Comment (tomat-suppe) [inline src/ChirpInfrastructure/CheepRepository.cs:37]: Do we need ToString()? Or can we remove it and order by timestamp? Just comment for later

File: src/ChirpWeb/Pages/UserTimeline.cshtml [hunk 4]
```diff
@@ -4,6 +4,10 @@
     ViewData["Title"] = "Chirp!";
     Layout = "Shared/_Layout";
     var routeName = HttpContext.GetRouteValue("author");
+
+    <!-- Pagination -->
```
Comment (HenrijetteOest) [inline src/ChirpWeb/Pages/UserTimeline.cshtml:8]: niceee, you found out to comment in html :))

**Review by HenrijetteOest — APPROVED** [review]
lgtm :)

File: src/ChirpInfrastructure/CheepRepository.cs (hunk 1 above)
Comment (JaSNOHo) [reply src/ChirpInfrastructure/CheepRepository.cs:38]: Not particularly, just for a better code structure. 
Just moved .OrderBy down to be above to .Skip and .Take

File: src/ChirpWeb/Pages/Public.cshtml [hunk 5]
```diff
@@ -30,4 +34,15 @@
     {
         <em>There are no cheeps so far.</em>
     }
+    <button class="button" onclick="navigateToPage(@previous)" style="color: ECCFC3;" id="previousButton"
```
Comment (tomat-suppe) [inline src/ChirpWeb/Pages/Public.cshtml:37]: Can this be moved to Layout class? Maybe then we can use it for both public and private

File: src/ChirpInfrastructure/CheepRepository.cs (hunk 2 above)
Comment (JaSNOHo) [reply src/ChirpInfrastructure/CheepRepository.cs:29]: Wasn't certain. Don't believe so.

**Review by tomat-suppe — APPROVED** [review]
Looks good! - I have added some comments for future consideration, so we can go back to this pull-request and decide whether we want to try to change these things.

**Review by llsa-itu — APPROVED** [review]
Looks great Jas, the variable names are really good.

**Review by Thyme-Cat — CHANGES_REQUESTED** [review]
One line break comment in CheepRepository

**Review by Thyme-Cat — APPROVED** [review]
lgtm now

**Pull request merged**